manifests/params.pp

```diff
@@ -6,25 +6,35 @@
       $package_name      = 'memcached'
       $package_provider  = undef
       $service_name      = 'memcached'
+      $service_path      = '/lib/systemd/system/'
```

Comment:
Is there a reason, why you're using /lib here? /etc seems better fitting for me (as it's not managed by a package) but maybe I'm missing something here?

Reply:
ups. I didn't do the rebase properly. fixed it